web/src/app/core/services/project.service.ts

```diff
@@ -25,8 +26,8 @@ export class ProjectService {
       access: { admin: [this.authService.profile.uid] },
       ...data,
       repositories: [],
-      createdOn: new Date(),
-      updatedOn: new Date(),
+      createdOn: firebase.firestore.Timestamp.fromDate(new Date()),
+      updatedOn: firebase.firestore.Timestamp.fromDate(new Date()),
```

Comment:
This might be easier to read if `declared` at the top under the imports?

```typescript
declare type Timestamp = firebase.firestore.Timestamp;
```

Then used with:


```typescript
createdOn: Timestamp.fromDate(new Date()),
updatedOn: Timestamp.fromDate(new Date()),
```

Reply:
@eddiejaoude 
 I have tried the same way as you suggested. But its giving me error "Timestamp is declared as type but is used as a value". Please see the attached screenshot for your reference.

![image](https://user-images.githubusercontent.com/10898595/58186144-f1e77e00-7cd1-11e9-9e3e-c16243d1c81f.png)

Reply:
Ah ok, yes it won't work, I was thinking of types not value 🤦‍♂ . Thank you for trying 👍